app/javascript/stylesheets/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import url('https://fonts.googleapis.com/css?family=Cairo:300,400,600|Lobster|Pacifico&display=swap&subset=cyrillic,cyrillic-ext');
```

Comment:
what the purpose of this file? don't you use the `application.scss` for the same things?